.gitpod.yml

```diff
@@ -3,16 +3,23 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - name: make
+  - name: ready
     command: |
       # Wait for docker to come up before doing make (make requires docker)
       while ! docker ps; do
         sleep 1
       done
       .githooks/linkallchecks.sh
-      make
   - name: docker_up
     command: sudo docker-up      
+  - prebuild: |
+      # Wait for docker to come up before doing make (make requires docker)
+      while ! docker ps; do
+        sleep 1
+      done
+      .githooks/linkallchecks.sh
+      make
```

Comment:
This is awesome. Gets the golang-build-container pulled in advance! Hopefully there aren't permissions problems in there.